src/results/steam_turbine.jl

```diff
@@ -30,16 +30,20 @@
 """
 `SteamTurbine` results keys:
 - `size_kw` Power capacity size [kW]
-- `year_one_thermal_consumption_mmbtu` Thermal (steam) consumption [MMBtu]
-- `year_one_electric_energy_produced_kwh` Electric energy produced in a year [kWh]
-- `year_one_thermal_energy_produced_mmbtu` Thermal energy produced in a year [MMBtu]
-- `year_one_thermal_consumption_series_mmbtu_per_hour` Thermal (steam) energy consumption series [MMBtu/hr]
-- `year_one_electric_production_series_kw` Electric power production series [kW]
-- `year_one_electric_to_grid_series_kw` Electric power exported to grid series [kW]
-- `year_one_electric_to_battery_series_kw` Electric power to charge the battery series [kW]
-- `year_one_electric_to_load_series_kw` Electric power to serve load series [kW]
-- `year_one_thermal_to_tes_series_mmbtu_per_hour` Thermal production to charge the HotThermalStorage series [MMBtu/hr]
-- `year_one_thermal_to_load_series_mmbtu_per_hour` Thermal production to serve the heating load SERVICES [MMBtu/hr]
+- `annual_thermal_consumption_mmbtu` Thermal (steam) consumption [MMBtu]
+- `annual_electric_production_kwh` Electric energy produced in a year [kWh]
+- `annual_thermal_production_mmbtu` Thermal energy produced in a year [MMBtu]
+- `thermal_consumption_series_mmbtu_per_hour` Thermal (steam) energy consumption series [MMBtu/hr]
+- `electric_production_series_kw` Electric power production series [kW]
+- `electric_to_grid_series_kw` Electric power exported to grid series [kW]
+- `electric_to_battery_series_kw` Electric power to charge the battery series [kW]
```

Comment:
Related to the comment above about "to_battery" vs "to_storage", while we do "storage_to" for Storage production: both SteamTurbine and CHP can produce electric to battery (ElectricStorage) and (hot) thermal to tes (HotThermalStorage). What we have now works, it's just a matter of simplifying everything to "storage", possibly at the expense of clarity about which storage we're talking about. I think it's fine as is, but just noting in case you think it should be changed.

Reply:
I did make the change to "to_storage" as you suggested and I think that will be okay for these techs that can go to both electric and thermal storage b/c it should be clear that "electric_to_storage" and "thermal_to_storage" are not the same "storage". I can also add a note in the help text

Reply:
They were mostly already clear but added a bit here https://github.com/NREL/REopt.jl/pull/160/commits/3ab961cd7fcd7a4418e540d4651b9b37b6de725f